### PR TITLE
Lazy loading for book covers

### PR DESCRIPTION
**Pull request by richard015ar**
Issue: https://github.com/pressbooks/pressbooks-book-directory-fe/issues/352

This change add [Vue LazyLoad](https://www.npmjs.com/package/vue-lazyload) library to add lazy loading feature to the book covers in the book cards. Until the images are loaded properly by scrolling, the default cover image is loaded.

**Note:** For testing, remember run `npm run install`

### CODE REVIEW

**Review by fdalcin — CHANGES_REQUESTED** [review]
@richard015ar PR looks good to me, thank you! 

I noticed that the `vue-lazyload` library works on error as well. Could we maybe replce `v-img-fallback` with the same one? That way we will be able to remove a dependency.

File: src/components/PbBookCover.vue [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <template>
-  <img
-    v-img-fallback="'/assets/images/default-book-cover.jpg'"
-    class="w-full"
-    :src="image"
+  <div
+    v-lazy-container="{ selector: 'img', error: defaulImg, loading: defaulImg }"
```
Comment (richard015ar) [inline src/components/PbBookCover.vue:3]: @fdalcin regarding your question, the error property does the job. I will remove the fallback lib dependency, you're right we won't need it anymore

Comment (fdalcin) [reply src/components/PbBookCover.vue:3]: We just need to replace the library on the collection cover I think

Comment (richard015ar) [reply src/components/PbBookCover.vue:3]: the collection cover doen't contain any fallback img

Comment (fdalcin) [reply src/components/PbBookCover.vue:3]: Yes, I saw that after that's why I deleted the message.

**Review by fdalcin — APPROVED** [review]

**Pull request merged**